src/index.html

```diff
@@ -5,8 +5,330 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <div class="wrapper">
+      <header class="header">
+        <div class="header__container">
+          <a href="#" class="header__logo">
+            <img src="./images/logo.png" alt="Moyo Logo">
+          </a>
+          <nav class="header__menu menu">
+            <ul class="menu__list">
+              <li class="menu__item">
+                <a href="#" class="menu__link is-active">Apple</a>
+              </li>
+              <li class="menu__item">
+                <a href="#" class="menu__link">Samsung</a>
+              </li>
+              <li class="menu__item">
+                <a href="#" class="menu__link">Smartphones</a>
+              </li>
+              <li class="menu__item">
+                <a href="#"
+                class="menu__link"
+                data-qa="nav-hover"
+                >
+                  Laptops & Computers
+                </a>
+              </li>
```

Comment:
you need to add the lines between the blocks, please check the checklist
<img width="1263" alt="Screenshot 2023-07-09 at 20 25 16" src="https://github.com/mate-academy/layout_catalog/assets/64746943/3032a8b8-ea0a-48d7-8b48-4c4cfa249383">